Source/RomanNumeral.h

```diff
@@ -12,6 +12,29 @@
  * @param input The number to convert.
  * @param output String buffer in which to store the converted Roman numeral.
  */
+
+typedef struct {
+    int num;
+    const char * romanNumeral;
+} TableEntry_t;
+
+const TableEntry_t romanNumeralTable[] = {
+    {1000, "M"},
+    {900, "CM"},
+    {500, "D"},
+    {400, "CD"},
+    {100, "C"},
+    {90, "XC"},
+    {50, "L"},
+    {40, "XL"},
+    {10, "X"},
+    {9, "IX"},
+    {5, "V"},
+    {4, "IV"},
+    {1, "I"}
+};
```

Comment:
The problem with declaring this here is that every module that includes this header will duplicate this structure and thus duplicate the memory allocation. Instead, you'll want to stick this into the C file and mark `romanNumeralTable` as `static` so that it's private to the module (not a global symbol) and so that it only gets allocated once.